styles/Microsoft/ComplexWords.yml

```diff
@@ -18,7 +18,7 @@ swap:
   accurate: right|exact
   acquiesce: agree
   acquire: get|buy
-  additional: more|extra
+  #additional: more|extra
```

Comment:
can we remove this instead of comment it out?

Reply:
This is me being non-committal and leaving the default MS styles there, in case we _do_ want to follow 100% of them